src/patternfly/_chart-globals.scss

```diff
@@ -76,16 +77,29 @@ $pf-chart-global--stroke-line-cap: "round";
 $pf-chart-global--stroke-line-join: "round";
 $pf-chart-global--stroke-dash-array: "10, 5";
 
+// Fills and Strokes
+$pf-chart-global--danger--Color--100: $pf-global--danger-color--100;
+$pf-chart-global--warning--Color--100: $pf-color-orange-300;
```

Comment:
Are there meant to be two variations of warning colors? @gdoyle1 looks like one is orange and one is gold.